src/services/FileStorageService.js

```diff
@@ -82,6 +82,8 @@ export const FileStorageServiceDataTypes = Object.freeze({
 	KML: 'application/vnd.google-earth.kml+xml'
 });
 
+export const TEMP_STORAGE_ID = 'temp_storage_id';
+
```

Comment:
Would suggest redoing this change: As before the temp. GeoResource ID just needs to be known locally